internal/pkg/exec/docker.go

```diff
@@ -159,3 +172,59 @@ func imageName(uri, tag string) string {
 	}
 	return fmt.Sprintf("%s:%s", uri, tag)
 }
+
+// IsEcrCredentialHelperEnabled return true if ecr-login is enabled either globally or registry level
+func (c DockerCommand) IsEcrCredentialHelperEnabled(uri string) bool {
+	// Make sure the program is able to obtain the home directory
+	splits := strings.Split(uri, "/")
+	if c.homePath != "" && len(splits) > 0 {
```

Comment:
nit: can we flip this if statement to reduce nesting
```go
if c.homePath == "" || len(splits) == 0 {
     return false
}
```

Reply:
Done